### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by Yazino12**
### In this Pull:

- Add React Redux
- Install react-redux
- Structure application file using Feature Folder approach using [ducks pattern](https://github.com/erikras/ducks-modular-redux)
- Write action creators and reducers for books
- Write action creators and reducers for categories
- Configure Redux Store and combine both reducers into a root reducer using the `cobmineRducers` function
- [Import redux methods correctly](https://github.com/Yazino12/bookstore-v2/pull/2/commits/167c58b0ddfdc71ad282734421cc08db75229f08)

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Redux from 'react-redux';
```
Comment (thecodechaser) [inline src/redux/configureStore.js]: - [x] Good job setting up the redux store but there are some issues you need to fix, You imported redux methods in the wrong way. If you will access your store in your components the application will throw errors and will not work. Please make sure to fix this issue and import redux methods in the correct way.
 - You can consider this code. `import { createStore, combineReducers } from 'redux';`

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Yazino12👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js (hunk 1 above)
Comment (Yazino12) [reply src/redux/configureStore.js]: @thecodechaser Thanks for the details in the review 🙏🏾. I will apply these changes right away.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Yazino12👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**